src/main.rs

```diff
@@ -96,7 +96,7 @@ fn main() -> io::Result<()> {
 
     robots::handle_robots(&content_dir, &output_path);
 
-    // Initialize Tera templates
+    // Intialize Tera templates
```

Comment:
```suggestion
    // Initialize Tera templates
```

Reply:
Can you believe I read it over and over and still didn't catch the error? My gosh...